www/src/pages/blog/astro-021-preview.astro

```diff
@@ -0,0 +1,119 @@
+---
+import { Markdown, Code } from 'astro/components';
+import BaseHead from '../../components/BaseHead.astro';
+import BlogHeader from '../../components/BlogHeader.astro';
+import BlogPost from '../../components/BlogPost.astro';
+import GoogleAnalytics from '../../components/GoogleAnalytics.astro';
+
+let title = 'Astro 0.21 Preview: Vite + WASM = ⚡️';
+let description = `Get a sneak preview of what's next for Astro, including our new Vite build engine and WASM-powered Go compiler.`;
+let publishDate = 'September 23, 2021';
+let permalink = 'https://astro.build/blog/astro-021-preview';
+let lang = 'en';
+---
+
+<html lang={ lang ?? 'en' }>
+  <head>
+    <BaseHead title={title} description={description} canonicalURL={permalink} image="https://astro.build/assets/blog/demo-day-2021-09/social.png" />
```

Comment:
@natemoo-re any chance that image you made for Matthew is available via Figma? I'm happy to make my own if I can borrow from that template